make/modules/jdk.jdwp.agent/Lib.gmk

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (c) 2011, 2020, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2011, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Same here, do not change Oracle copyright

Reply:
Done